JWT/CWT claims and CBOR tag for CMWs (#32)
thomas-fossati/draft-ftbs-rats-msg-wrap: draft-ftbs-rats-msg-wrap.md

The other format wraps the value in a CBOR byte string and prepends a
CBOR tag to convey the type information.

> An associated CBOR tag, as well as JWT and CWT claims are also defined to allow embedding these formats into CBOR-based protocols and web tokens.

Reviewer: ```suggestion
This document defines a corresponding CBOR tag as well as JWT and CWT claims allow embedding these formats into CBOR-based protocols and web tokens.
```

Reviewer: Maybe:
formats -> data structures?

Reviewer 2: I have updated your suggestion:
* to fix some mismatched grammar, and
* change "formats" into "wrapped conceptual messages"

Author: Maybe say "...into CBOR-based protocols and structures, e.g., web tokens."

Reviewer: :sparkles: lgtm

The other format wraps the value in a CBOR byte string and prepends a
CBOR tag to convey the type information.

> This document defines a corresponding CBOR tag, as well as JWT and CWT claims.  These allow embedding the wrapped conceptual messages into CBOR-based protocols and web tokens.

Author: ```suggestion
This document defines a corresponding CBOR tag, as well as JWT and CWT claims.  These allow embedding the wrapped conceptual messages into CBOR-based protocols and structures, e.g., web tokens.
```

Reviewer 2: Ned, the problem with the suggestion is that JWT is not a CBOR-based structure.

The two sentences are structured symmetrically:
* CBOR tag -> CBOR-based protocols
* JWT and CTW claims -> web tokens

Maybe we should add a "respectively" to make the connection more explicit?

Author: OK

Reviewer 2: see b288559f5c0e6e506ef81fa8af5bd233756a707e
